context/amf_ue.go

```diff
@@ -147,9 +147,9 @@ type AmfUe struct {
 	SmContextList sync.Map `json:"-"` // map[int32]*SmContext, pdu session id as key
 	/* Related Context*/
 	//RanUe map[models.AccessType]*RanUe `json:"ranUe,omitempty" yaml:"ranUe" bson:"ranUe,omitempty"`
-	RanUe map[models.AccessType]*RanUe `json:"ranUe, omitEmpty"`
+	RanUe map[models.AccessType]*RanUe `json:"ranUe,omitEmpty"`
 	/* other */
-	onGoing                       map[models.AccessType]*OnGoing        `json:"onGoing,omitempty"`
+	onGoing                       map[models.AccessType]*OnGoing
```

Comment:
Why not doing the following (with the corresponding changes in the respective files)?
```suggestion
	OnGoing                       map[models.AccessType]*OnGoing        `json:"onGoing,omitempty"`
```

Reply:
This one also is problematic because a struct and a method already exist with that name. I will try to rename them and see how this goes.